src/main/java/dev/openfeature/sdk/OpenFeatureAPI.java

```diff
@@ -131,14 +131,14 @@ public void setProvider(String clientName, FeatureProvider provider) {
     /**
      * Set the default provider and wait for initialization to finish.
      */
-    public void setProviderAndWait(FeatureProvider provider) {
+    public void setProviderAndWait(FeatureProvider provider) throws RuntimeException {
```

Comment:
`RuntimeException` is unchecked, so this is non-breaking.

OpenFeature Errors are all subclasses unchecked, so this is no problem for providers who want to throw OpenFeature errors.